### PR TITLE
Fix macOS toolchain

### PR DESCRIPTION
**Pull request by cnorthrop**
Didn't realize this was broken.  It is required for vktrace.

### CODE REVIEW

File: update_external_sources.sh [hunk 1]
```diff
@@ -3,7 +3,17 @@
 
 set -e
 
-CURRENT_DIR="$(dirname "$(readlink -f ${BASH_SOURCE[0]})")"
+CORE_COUNT=0
```
Comment (stroyan) [inline update_external_sources.sh]: 1 seems like a safe initial value.  I don't see a defined behavior for 'make -j 0'

Comment (cnorthrop) [inline update_external_sources.sh]: Just trying to make it really obvious that something is wrong.  Probably leaving it undefined would work better, although that really buries some systems.

**Comment by cnorthrop** [comment]
Updated to have sensible default CORE_COUNT.

**Comment by cnorthrop** [comment]
cherry-picked to master

**Pull request closed without merging**